x-pack/plugins/fleet/tsconfig.json

```diff
@@ -108,5 +108,6 @@
     "@kbn/zod-helpers",
     "@kbn/react-kibana-mount",
     "@kbn/react-kibana-context-render",
+    "@kbn/fleet-plugin",
```

Comment:
should be removed

Reply:
This keeps getting added by kibanamachine
this is the log message
```
Author: kibanamachine <42973632+kibanamachine@users.noreply.github.com>
Date:   Fri May 24 06:56:49 2024 +0000

    [CI] Auto-commit changed files from 'node scripts/lint_ts_projects --fix'
```